From: Steve Dickson <steved@redhat.com>
Date: Thu, 27 Jul 2017 10:35:55 -0400
Subject: mountstats:  Remove a shebang
Origin: https://git.linux-nfs.org/?p=steved/nfs-utils.git;a=commit;h=caa59a30b78dedea99a2317da95bb86810360645

There might be an issue that the script is executed with unwanted
version of <lang>.

Signed-off-by: Steve Dickson <steved@redhat.com>
---
 tools/mountstats/mountstats.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/mountstats/mountstats.py b/tools/mountstats/mountstats.py
index 88ccdae78c9f..a68d70225769 100644
--- a/tools/mountstats/mountstats.py
+++ b/tools/mountstats/mountstats.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python
 # -*- python-mode -*-
 """Parse /proc/self/mountstats and display it in human readable form
 """
-- 
2.32.0

